contributions/executable-tutorial/charande-lazarc/README.md

```diff
@@ -0,0 +1,27 @@
+# GitLab CI/CD-server implementation  
+Category: Executable tutorial
+## Authors
+
+Name: [Lazar Cerovic](https://github.com/lazarko)
+
+
+E-mail: lazarc@kth.se
+
+
+Name: [Charlotte Andersson](https://github.com/charlottea98)
+
+
+E- mail: charande@kth.se
+
+## Relevance to DevOps
+Continuous integration (CI) and continuous deployment/delivery (CD) are fundamental
+tools for DevOps development. CI is an automated way for continuously merging and testing
+software code, while CD is by continuously make deliveries of software functionality in so-called deployments. 
+The difference between continuous deployment and delivery is that continuous deployment is actually 
+deployed, while continuous delivery is not. 
+
+## Originality
+
+There are no awesome tutorials on Katacoda that illustrates how to create a CI/CD on GitLab. 
+Although, CI/CD are common tools that are used by many software developers, along with many
+types of DevOps tools. 
```

Comment:
Can you be more specific. Which tech stack are you going to use for the CI/CD? Which technology specifically?

Reply:
I do not understand is it not specified as GitLab? Or is it not counted as Tech Stack?